_posts/2023-06-11-ayb-helping-you-own-your-data.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title: 'ayb: Helping you own your data'
+date: 2023-06-11 22:45 +0000
+---
+## Introduction
+
+Today's databases are ubiquitous, but frustratingly inaccessible to most people. Your own data likely lives in thousands of databases at various companies and organizations, but if you wanted to create a database for yourself in which to store data and share it, you need the skills of both a system administrator and a software engineer. By virtue of the complexity of database management systems (and market forces), your data either lives in other people's databases or outside of a database in text files and spreadsheets.
+
+Three groups of people in particular feel the pain of databases being hard to set up: students that are just getting started, sharers like journalists and scientist who are experts in fields other than hosting databases, and anyone who wants to have sovereignty over their data but isn't sure how to get started. We'll dive into each of these groups more in [students, sharers, and sovereigns](#students-sharers-and-sovereigns).
+
+I've been working on a project called [ayb](https://github.com/marcua/ayb), which is a relational database management system that makes it easier to to create databases, share them with collaborators, and query them from a web application or the command line. With `ayb`, [all your base can finally belong to you](https://www.youtube.com/watch?v=qItugh-fFgg). What does `ayb` offer?
```

Comment:
Can you use gitpod or something to add a button that launches an instance of ayb? Then the steps to try out are fork, click button, use in a page

Reply:
This is an awesome idea. I'll try to bring it into a v2!